src/pudl/transform/classes.py

```diff
@@ -782,7 +852,14 @@ class AbstractTableTransformer(ABC):
     """Cached intermediate dataframes for use in development and debugging."""
 
     parameter_model = TableTransformParams
+    """The :mod:`pydantic` model that is used to contain & instantiate parameters.
+
+    In child classes this should be replaced with the data source-specific
+    :class:`TableTransformParams` class, if it has been defined.
+    """
```

Comment:
this is the first time you are using the language of "model" here. idk if that is bc it is applicable here and not other places but I would suggest being consistent if possible. idk what the word "model" does here in conveying information so I would suggest removing to mirror the class name

Reply:
Pydantic calls its classes (data) [models](https://pydantic-docs.helpmanual.io/usage/models/). I'll try and be more uniform with this.